Add more 64 bits arches to the list of arches where pointers are longs,
otherwise strings gets typedef'd to an int as p_int and breaks in conf_mg.c.

Index: mgetty.h
--- mgetty.h.orig
+++ mgetty.h
@@ -218,7 +218,9 @@ typedef unsigned char uch;
  * union to use "void *" and "long", instead of "int" (see config.h).
  * Same for Sparc Ultra machines [at least with SparcLinux]
  */
-#if defined(__alpha__) || defined(__sparc64__) || \
+#if defined(_LP64)
+# define PTR_IS_LONG
+#elif defined(__alpha__) || defined(__sparc64__) || \
 	    defined(__ia64__) || defined(__s390x__) || defined(__x86_64__) || \
 	    defined(__powerpc64__)
 # define PTR_IS_LONG
